src/app_engine/gcm_notify.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python2.4
+# Copyright 2015 Google Inc. All Rights Reserved.
+"""Module for communication with GCM server.
```

Comment:
unit test? We can stub out the gcm API in a similar way as analytics_test. Probably moving the mocking code test_unitilities.